### PR TITLE
fixed dimuon invariant mass plots range

### PR DESCRIPTION
**Pull request by llunerti**
#### PR description:

This PR fixes the range of the following histograms under the Muons/diMuonHistograms area of the Offline DQM:
- LooseLooseMuon
- LooseLooseMuon_Barrel
- LooseLooseMuon_EndCap
- MediumMediumMuon 
- MediumMediumMuon_Barrel
- MediumMediumMuon_EndCap 
- TightTightMuon 
- TightTightMuon_Barrel
- TightTightMuon_EndCap 

The ranges of all the above plots were not centered on the Z peak, so they were showing no entries (e.g. https://tinyurl.com/ytbrrk9w). To fix this I've just put the correct mass range around the Z peak (namely from `HighMassMin` to `HighMassMax` defined [here](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Muon/python/diMuonHistograms_cfi.py#L22-L23)).

#### PR validation:
I have checked that the above plots are filled as intended by running the `10842.0` Z->MuMu workflow on CMSSW_13_2_6. The Z peak was visible as expected.

FYI @caruta 

P.S. This PR was opened by mistake in the main branch: [https://github.com/cms-sw/cmssw/pull/43026](https://github.com/cms-sw/cmssw/pull/43026)

### CODE REVIEW

File: DQMOffline/Muon/src/DiMuonHistograms.cc
Comment (rappoccio) [inline DQMOffline/Muon/src/DiMuonHistograms.cc:118]: These need to be `highMassMin`, `highMassMax`.

**Review by rappoccio — CHANGES_REQUESTED** [review]

File: DQMOffline/Muon/src/DiMuonHistograms.cc
Comment (llunerti) [reply DQMOffline/Muon/src/DiMuonHistograms.cc:118]: I see, but this means changing their definition [here](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Muon/python/diMuonHistograms_cfi.py#L20-L23). The same would also apply to `LowMassMin` and `LowMassMax`, and this has nothing to do with the issue I would like to fix. Should I go ahead and fix the variables name as well?

Comment (rappoccio) [reply DQMOffline/Muon/src/DiMuonHistograms.cc:118]: Yes, please fix this. Thanks!

Comment (tjavaid) [reply DQMOffline/Muon/src/DiMuonHistograms.cc:118]: @llunerti , could you please follow up on this?

**Comment by llunerti** [comment]
sorry @rappoccio re-requested for review by mistake...

**Review by rappoccio — APPROVED** [review]

**Comment by tjavaid** [comment]
+1

**Comment by rappoccio** [comment]
hold

* Holding until 14_0_X is created, so we do not merge it by accident.

**Comment by llunerti** [comment]
@rappoccio [here it is](https://github.com/cms-sw/cmssw/pull/43626), sorry for the delay!

**Comment by antoniovilela** [comment]
unhold

**Comment by antoniovilela** [comment]
backport of #43626

**Comment by antoniovilela** [comment]
please test

**Comment by antoniovilela** [comment]
please test

**Comment by tjavaid** [comment]
+1

**Comment by antoniovilela** [comment]
+1

**Pull request merged**
